src/test/java/com/list/todo/util/ObjectsProvider.java

```diff
@@ -192,4 +202,25 @@ public static TagInput getTagInput() {
         }};
     }
 
+    public static User createUser(UserInput userInput) {
+        return User.builder()
+                .name(userInput.getUsername())
+                .username(userInput.getUsername())
+                .email(userInput.getEmail())
+                .password(new BCryptPasswordEncoder().encode(userInput.getPassword()))
+                .gravatarHash("ffg")
+                .role(RoleName.ROLE_USER)
+                .build();
+    }
+
+    public static List<User> createListOfFollowers(UserSettings userSettings) {
+        User user1 = createUserWithUserSettings(1, userSettings);
+        User user2 = createUserWithUserSettings(2, userSettings);
+
+        return new ArrayList<User>() {{
```

Comment:
I'd recommend to avoid using this kind of initialization. Under hood it creates anonymous subclass, which may lead to unexpected results. I personally have spent considerable amount of time trying to figure out why my unit tests were failing, and the reason was such initialization.
In this concrete case use Arrays.asList(). To initialize map or set just create it and then .put() or .add() values.

Reply:
Also you can move to java 11, and use convenient factory methods Set.of(), Map.of()